tox.ini

```diff
@@ -4,11 +4,13 @@ envlist = py37, py38, py39, py310, lint
 skipsdist = True
 
 [testenv]
+usedevelop=True
```

Comment:
Is this necessary? When I take it, docs test passed some errors (but finally failed with sphinx errors...)